### ISSUE TITLE
Update intervention/image version

### ISSUE THREAD

**Issue by broqit** [simonhamp/the-og#29]
Vendor intervention/image needs to be updated to min 3.5.0 ver.

**Comment by simonhamp** [comment]
What's your issue?

**Comment by ladaposamuel** [comment]
I think it might be related to this error when you run `composer require simonhamp/the-og`;

I am currently facing a blocker installing on a Laravel `"laravel/framework": "^9.19",`

```bash
  
./composer.json has been updated
Running composer update simonhamp/the-og
Loading composer repositories with package information
Updating dependencies                                 
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - simonhamp/the-og[v0.1.0, ..., v0.5.4] require intervention/image ^3.1 -> found intervention/image[3.1.0, ..., 3.7.2] but the package is fixed to 2.7.2 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - simonhamp/the-og v0.5.5 requires intervention/image ~3.1.0 -> found intervention/image[3.1.0] but the package is fixed to 2.7.2 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - simonhamp/the-og v0.5.6 requires intervention/image 3.3.3 -> found intervention/image[3.3.3] but the package is fixed to 2.7.2 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires simonhamp/the-og * -> satisfiable by simonhamp/the-og[v0.1.0, ..., v0.5.6].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
You can also try re-running composer require with an explicit version constraint, e.g. "composer require simonhamp/the-og:*" to figure out if any version is installable, or "composer require simonhamp/the-og:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.

```
